File: rails/app/models/county.rb
class County < ApplicationRecord
  self.primary_key = 'id'
  belongs_to :rpa
  has_many :places
  include PgSearch::Model
  include ActiveModel::Dirty
  pg_search_scope :search_by_county, against: [:county], using: { tsearch: { any_word: true } }
  

 
end

File: ember/app/models/county.js
import DS from 'ember-data';
import { attr, belongsTo, hasMany } from '@ember-decorators/data';


export default class extends DS.Model {
  @belongsTo('rpa', {async: true}) rpa
  @hasMany('place', { async: true }) places

  @attr('string') geoid
  @attr('string') county
  @attr('string') namelsad
  @attr('boolean' , { default: true }) ispublic
  @attr('boolean', { default: false }) isShowingCities
  
}
